setup_dev_container.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SCRIPT_DIR="$SOURCE_DIR/.travis"
+pushd $SCRIPT_DIR
+
+source $SCRIPT_DIR/travis-common.inc
+
+set -e
+set -x
+
+HOME=${HOME-/}
+
+release=$($SCRIPT_DIR/get_rawhide_version.py)
+image=fedora/fedora:$release-$(uname -m)
+
+overridesdir=$(python3 -c 'import gi; print(gi._overridesdir)')
+
+mmd_setup_container \
+    os=fedora \
+    release=$release \
+    repository=quay.io \
+    image=$image \
+    oci_extra_commands="RUN ln -sf /builddir/bindings/python/gi/overrides/Modulemd.py ${overridesdir}/Modulemd.py" \
```

Comment:
```suggestion
    oci_extra_commands='RUN ln -sf /builddir/bindings/python/gi/overrides/Modulemd.py $(python3 -c "import gi; print(gi._overridesdir)")/Modulemd.py' \
```
The overrides directory needs to be the one _inside_ the container. (And drop the `overridesdir=...` line above.)

Reply:
Another great catch. Thanks!